library/src/main/kotlin/com/livefront/gsonkotlinadapter/KoltinReflectiveTypeAdapterFactory.kt

```diff
@@ -107,6 +122,10 @@ class KotlinReflectiveTypeAdapterFactory private constructor() : TypeAdapterFact
             }
             reader.endObject()
 
+            // Add all default values if the JSON did not include it
```

Comment:
Same thing here...this isn't "all default values", it's just the primitive ones. Can we make this comment more specific?

Reply:
See answer above.